Day-3-Exercises/main.js

```diff
@@ -0,0 +1,126 @@
+//Exercises: Level 1
+
+//Declare firstName, lastName, country, city, age, isMarried, year variable and assign value to it and use the typeof operator to check different data types.
+let firstName = 'Dmitry'
+let lastName = 'Lazarev'
+let country = 'Ukraine'
+let city = 'Odessa'
+let age = 44
+let isMarried = true
+let year = 2024
+console.log(typeof(firstName), '\n', typeof(lastName),'\n', typeof(country),'\n', typeof(city),'\n', typeof(age),'\n', typeof(isMarried),'\n', typeof(year))
+
+//Check if type of '10' is equal to 10
+console.log('10' === 10)
+
+//Check if parseInt('9.8') is equal to 10
+console.log(parseInt('9.8') == 10)
+
+/* Boolean value is either true or false.
+Write three JavaScript statement which provide truthy value.
+Write three JavaScript statement which provide falsy value. */
+let undef
+console.log(5>4, '10' == 10, 'text' == 'text')
+console.log(10 <=5 && 5>3, 5-3==1, 'Text' == 'text')
+
+/*Figure out the result of the following comparison expression first without using console.log(). After you decide the result confirm it using console.log()
+
+4 > 3 (true)
+4 >= 3 (true)
+4 < 3 (false)
+4 <= 3 (false)
+4 == 4 (true)
+4 === 4 (true)
+4 != 4 (false)
+4 !== 4 (false)
+4 != '4' (false)
+4 == '4' (true)
+4 === '4' (false)
+Find the length of python and jargon and make a falsy comparison statement. */
+
+/*Figure out the result of the following expressions first without using console.log(). After you decide the result confirm it by using console.log()
+
+4 > 3 && 10 < 12 (true)
+4 > 3 && 10 > 12 (false)
+4 > 3 || 10 < 12 (true)
+4 > 3 || 10 > 12 (true)
+!(4 > 3) (false)
+!(4 < 3) (true)
+!(false) (true)
+!(4 > 3 && 10 < 12) (fale)
+!(4 > 3 && 10 > 12) (true)
+!(4 === '4') (true)
+There is no 'on' in both dragon and python (logiacly false but this is nither arithmetic not logic operation) */
+
+
+/*Use the Date object to do the following activities
+
+What is the year today?
+What is the month today as a number?
+What is the date today?
+What is the day today as a number?
+What is the hours now?
+What is the minutes now?
+Find out the numbers of seconds elapsed from January 1, 1970 to now. */
+const myDateObj = new Date()
+console.log('The year:', myDateObj.getFullYear())
+console.log('The month:', myDateObj.getMonth())
+console.log('The date:', myDateObj.getDate())
+console.log('The day:', myDateObj.getDay)
```

Comment:
You should call `.getDay` method by opening parentheses.

Reply:
fixed